### PR TITLE
[1LP][RFR] ignore test_sdn_balancers_tagvis for 5.11

### PR DESCRIPTION
**Pull request by lina-nikiforova**
## Purpose or Intent
Load balancers are no longer there

### CODE REVIEW

File: cfme/tests/networks/test_sdn_balancers.py [hunk 1]
```diff
@@ -68,6 +68,7 @@ def test_sdn_balancers_detail(provider, network_prov_with_load_balancers):
 
 
 # only one provider is needed for that test, used Azure as it has balancers
+@pytest.mark.ignore_stream("5.11", "upstream")  # BZ 1672949 (RFE)
```
Comment (kedark3) [inline cfme/tests/networks/test_sdn_balancers.py]: Is that only test you need to ignore or all the `test_sdn_balancers`?

Comment (lina-nikiforova) [reply cfme/tests/networks/test_sdn_balancers.py]: @kedark3 good catch! it's the whole module that needs to be blocked

Comment (lina-nikiforova) [reply cfme/tests/networks/test_sdn_balancers.py]: updated

Comment (kedark3) [reply cfme/tests/networks/test_sdn_balancers.py]: Awesome I approve.

**Comment by lina-nikiforova** [comment]
@digitronik added. Not sure how it is used in Polarion though. We can continue the discussion offline.

**Comment by sbulage** [comment]
> @digitronik added. Not sure how it is used in Polarion though. We can continue the discussion offline.

Nice question @lina-nikiforova, by adding `endsin` in the test case will ensure any user that this test case/feature not supported in upcoming CFME versions. We were using `endsin` to generate Test Runs for respective CFME version.

**Pull request merged**